src/activate/activate.c

```diff
@@ -82,7 +82,7 @@ static int open_sockets(int *epoll_fd, bool accept) {
          */
 
         STRV_FOREACH(address, arg_listen) {
-                fd = make_socket_fd(LOG_DEBUG, *address, arg_socket_type, (arg_accept*SOCK_CLOEXEC));
+                fd = make_socket_fd(LOG_DEBUG, *address, arg_socket_type, (arg_accept * SOCK_CLOEXEC));
```

Comment:
I'm not sure that is much better. Dropping the parens would be more useful.

Reply:
ok, needs to be done manually...